packages/auth-browser/src/lib/chains/sol.ts

```diff
@@ -105,9 +106,11 @@ export const checkAndSignSolAuthMessage = async (): Promise<AuthSig> => {
 
   try {
     // when it's not in local storage, it's a string
+    // @ts-ignore
```

Comment:
We probably want to refactor the IEither type at some point so we don't have to do this